### PR TITLE
Add outline support to Alert component

### PR DESCRIPTION
**Pull request by HendrikGrobler**
The button component supports an outline display mode and users may want to display alerts this way for certain designs.

Docs PR: https://github.com/vuetifyjs/docs/pull/353

screenshot_20171030_144202

### CODE REVIEW

**Comment by HendrikGrobler** [comment]
Added docs PR.

File: src/components/VAlert/VAlert.js [hunk 1]
```diff
@@ -38,10 +39,12 @@ export default {
...
     classes () {
       const colorProp = (this.type && !this.color) ? 'type' : 'computedColor'
-
-      return this.addBackgroundColorClassChecks({
-        'alert--dismissible': this.dismissible
-      }, colorProp)
+      const classes = {
+        'alert--dismissible': this.dismissible,
+        'alert--outline': this.outline
+      }
```
Comment (johnleider) [inline src/components/VAlert/VAlert.js]: add an empty line

File: src/stylus/components/_alerts.styl [hunk 2]
```diff
@@ -17,6 +17,8 @@
     align-self: center
     color: $alert-icon-color
     font-size: $alert-icon-font-size
+  &--outline .icon
+    color: currentColor !important
```
Comment (johnleider) [inline src/stylus/components/_alerts.styl]: Can this be set to just inherit the color and ditch the **!important**?

File: src/components/VAlert/VAlert.js (hunk 1 above)
Comment (HendrikGrobler) [reply src/components/VAlert/VAlert.js]: Done.

File: src/stylus/components/_alerts.styl (hunk 2 above)
Comment (HendrikGrobler) [reply src/stylus/components/_alerts.styl]: I'll check and confirm, but from what I remember !important was needed. By inherit do you mean still using `currentColor` or something like `color: inherit`?

Comment (johnleider) [reply src/stylus/components/_alerts.styl]: Either if they will work without important. If it's needed though, just use `color: inherit`

Comment (HendrikGrobler) [reply src/stylus/components/_alerts.styl]: Both and `inherit` and `currentColor` don't work without `!Important` (they appear grey). Changing this to use `color: inherit !Important` then.

**Pull request merged**